onyxia-api/src/main/java/fr/insee/onyxia/api/controller/pub/HealthcheckController.java

```diff
@@ -0,0 +1,42 @@
+package fr.insee.onyxia.api.controller.pub;
+
+import fr.insee.onyxia.api.configuration.CatalogWrapper;
+import fr.insee.onyxia.api.configuration.Catalogs;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@Tag(name = "Public")
+@RequestMapping("/public")
+public class HealthcheckController {
```

Comment:
Side topic: I guess with this Onyxia also takes an position to move away from Kotlin (and the very limited part of kotlin code that exists) towards Java? If yes kotlin can probably be cleaned up from the build config, just so collaborators don't have to be unsure which language to use.